modules/aspect_bazel_lib/1.21.2/MODULE.bazel

```diff
@@ -0,0 +1,23 @@
+"aspect-build/bazel-lib"
+
+module(
+    name = "aspect_bazel_lib",
+    compatibility_level = 1,
+    version = "1.21.2",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.3.0")
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "stardoc", repo_name = "io_bazel_stardoc", version = "0.5.0")
+
+ext = use_extension("@aspect_bazel_lib//lib:extensions.bzl", "ext")
+
+use_repo(ext, "jq_toolchains")
+
+use_repo(ext, "yq_toolchains")
+
+register_toolchains(
+    "@jq_toolchains//:all",
```

Comment:
@gregmagolan should the new copy directory ones be here too?

Reply:
not yet; that will come in a https://github.com/aspect-build/bazel-lib/pull/321; this release was just for the tool binary release